canonical/react-components
# feat: upstream FormikField

## Done

- upstream the FormikField component to make react-component's fields and Formik easier to work with. This component is already in use in maas-ui and react-components and we want to use it in another project.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Go to the FormikField docs page in Storybook.
- Check that the docs make sense.

## Fixes

https://warthogs.atlassian.net/browse/WD-9813
Fixes: #957.
